src/pages/contribute.tsx

```diff
@@ -14,16 +14,16 @@ export default function Contribute() {
     <Layout title='How to Contribute'>
       <Hero title='How to Contribute'>
         <p className='hero__text'>
-          Jellyfin is a community project run by volunteers. We&apos;re always looking for additional help.
+          Jellyfin is a community project run by volunteers.\n We are always looking for additional help.
```

Comment:
Sorry I thought I had commented on this already... maybe it would make more sense to use `<br />` here for the line break?

```suggestion
          Jellyfin is a community project run by volunteers.
          <br />
          We are always looking for additional help.
```